examples/demo_lfr.ipynb

```diff
@@ -30,9 +30,13 @@
     "from sklearn.linear_model import LogisticRegression\n",
     "from sklearn.preprocessing import StandardScaler\n",
     "from sklearn.metrics import accuracy_score\n",
+    "from sklearn.metrics import classification_report\n",
```

Comment:
Please bring the reference to the top of the notebook

Reply:
ok